Vise Clave-logo fra svg. Drar inn bibliotek: gatsby-plugin-react-svg

evilppo opened the pull request:
Ingen animasjoner enda.

Commit:
Vise Clave-logo fra svg. Drar inn bibliotek: gatsby-plugin-react-svg

Ingen animasjoner enda.

evilppo commented:
Fargen på logo satt til svart for å være mer synlig frem til vi får på plass animering

peteabre reviewed (changes requested):
Er noe med linkene som er blitt endret etter at logoen ble introdusert. Se screenshot
<img width="544" alt="Skjermbilde 2019-10-09 kl  07 54 23" src="https://user-images.githubusercontent.com/4180788/66455160-42d1f600-ea6a-11e9-932c-5fb78cacee69.png">

Commit:
Fjerne 'height'-prop. Dette stopper 'Wrapper' fra å bli høyere enn nødvendig.

 Fikser dermed buggen som gir for stort mellomrom mellom ClaveLink og 'border bottom'

evilppo commented:
> Er noe med linkene som er blitt endret etter at logoen ble introdusert. Se screenshot
> <img alt="Skjermbilde 2019-10-09 kl 07 54 23" width="544" src="https://user-images.githubusercontent.com/4180788/66455160-42d1f600-ea6a-11e9-932c-5fb78cacee69.png">

@peteabre : Fikset i: 9dfa868
